package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.5.10",
```

Comment:
Removed and installed graphql-request to store cache instead, because couldn't figure out how to correctly get the cache to display properly on mutations